web/src/engine/websites/templates/FlatManga.ts

```diff
@@ -0,0 +1,110 @@
+﻿import { AddAntiScrapingDetection, FetchRedirection } from "../../platform/AntiScrapingDetection";
+import { FetchCSS } from "../../platform/FetchProvider";
+import { DecoratableMangaScraper, type Chapter, type MangaScraper } from "../../providers/MangaPlugin";
+import { Page } from "../../providers/MangaPlugin";
+import * as Common from '../decorators/Common';
+
+AddAntiScrapingDetection(async (invoke) => {
+    const result = await invoke<boolean>(`document.documentElement.innerHTML.includes('ct_anti_ddos_key')`);// Sample => Mangagun, NicoManga, Rawinu, Weloma, WeloveManga
```

Comment:
Prefer CSS selector instead of matching text since this would be less error prone